src/styles/App.css

```diff
@@ -1,5 +1,7 @@
 body {
   margin: auto;
-  width: 80%;
+  width: 100%;
```

Comment:
I would not use the App.css file for now, and would rather go with e.g. w-screen / h-screen or the full options in the components themselves. maybe we will need another file in the upcoming week, but for readability, there should not be to many files with just a few style changes.

Reply:
ooh we can take that out completely, we arent using any styles from there

Reply:
we would remove unused files during refactoring. I will add it to our todos for next week so we remember to assign it.👍🏻👍🏻